Algorithms.Test/ShuffleTheArrayTest.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Xunit;
+
+namespace Algorithms.Test
+{
+    public class ShuffleTheArrayTest
+    {
+        private ShuffleTheArray _shuffleTheArray;
+        /// <summary>
+        /// Constructor
+        /// </summary>
+        public ShuffleTheArrayTest() {
+            _shuffleTheArray    = new ShuffleTheArray();
+        }
+
+        [Fact]
+        public void TestCase1() {
+            //Input
+            int[] numsArray     = new int[] { 2, 5, 1, 3, 4, 7 };
```

Comment:
adding the spaces between the assignation is generaly not a good practice, since adding a new line requires adjustment on each line

Reply:
Got it!. I read once, that my code should be easy to read, like a book. I assumed if I used tabs or extra space. It would achieve the goal. I think ... i didn't understand well hehe

Reply:
That's true, it should be easy to read as a book. One part of that is to use spaces, but in a different way for example:
What's easier to read, this:
```js
let y = 10;
let x = 1;
if(y==='division')
return x/y;
else if(y==='mult')
return x*y;
return x+y;
```

or this:
```js
let operator = 10;
let myNumber = 1;

if (operation === 'division')
   return myNumber/operator;
else if (operation === 'mult')
  return myNumber*operator;

return myNumber+operator;
```

If you see, I changed 2 things, one was spaces/tabs and the second was variable naming.
So yes, that's true adding spaces helps, but you should think also in maintainability, for example

```js
let x          = true
let y          = false
let z          = 1 
```

and what happen if you want to add a variable called `superLongName`:
```js
let x          = true
let y          = false
let z          = 1 
let superLongName = true
```

Then you have either of 2 problems, they are not aligned anymore, so not easy to read, or you need to update 3 lines instead of just adding your variable.

Reply:
Great!! I definitely didn't get it right. Now i see the problem 👏